docs/use.md

```diff
@@ -0,0 +1,180 @@
+        * `disk_type` (for 2nd gen, defaults to ssd)
+        * `failover_replica_name` (only for 2nd gen, if specified creates a failover replica, defaults to "")
+        * `maintenance_window_day` (for 2nd gen only, defaults to 1 (Sunday))
+        * `maintenance_window_hour` (for 2nd gen only, defaults to 0)
+        * `backups_enabled` (defaults to true, set to "false" to disable)
+        * `backup_start_time` (defaults to 06:00)
+        * `binlog` (defaults to false for 1st gen, true for 2nd gen, set to "true" to use)
+        * `activation_policy` (defaults to on demand)
+        * `replication_type` (defaults to synchronous)
+        * `auto_resize` (2nd gen only, defaults to false, set to "true" to use)
+    * Bind
+        * `username` (defaults to a generated value)
+        * `password` (defaults to a generated value)
+
+        **Example Binding credentials**
+
+        ```
+        "credentials": {
+             "CaCert": "-----BEGIN CERTIFICATE-----\nredacted\n-----END CERTIFICATE-----",
+             "ClientCert": "-----BEGIN CERTIFICATE-----\nredacted\n-----END CERTIFICATE-----",
+             "ClientKey": "-----BEGIN RSA PRIVATE KEY-----\redacted\n-----END RSA PRIVATE KEY-----",
+             "Password": "unencoded-redacted",
+             "Sha1Fingerprint": "redacted",
+             "Username": "redacted",
+             "database_name": "redacted",
+             "host": "255.255.255.255",
+             "instance_name": "redacted",
+             "last_master_operation_id": "some-guid",
+             "uri": "mysql://username:encodedpassword@host/databasename?ssl_mode=required"
+        }
+        ```
+
+* [ML APIs](https://cloud.google.com/ml/)
+    * Bind
+        * `role` without "roles/" prefix (see https://cloud.google.com/iam/docs/understanding-roles for available roles)
+
+        **Example Binding credentials**
+
+        ```
+        "credentials": {
+             "Email": "redacted",
+             "Name": "redacted",
+             "PrivateKeyData": "redacted",
+             "UniqueId": "redacted",
+        }
+        ```
+
+* [Bigtable](https://cloud.google.com/bigtable/docs/)
+    * Provison
+        * `name` (defaults to a generated value)
+        * `cluster_id` (defaults to a generated value)
+        * `display_name` (defaults to a generated value)
+        * `storage_type` (one of "SSD" or "HDD", defaults to "SSD")
+        * `zone` (defaults to us-east1-b)
+        * `num_nodes` (defaults to 3)
+    * Bind
+        * `role` without "roles/" prefix (see https://cloud.google.com/iam/docs/understanding-roles for available roles), e.g. editor
+
+        **Example Binding credentials**
+
+        ```
+        "credentials": {
+             "Email": "redacted",
+             "Name": "redacted",
+             "PrivateKeyData": "redacted",
+             "UniqueId": "redacted",
+             "instance_id": "foobar",
+        }
+        ```
+
+* [Spanner](https://cloud.google.com/spanner/docs/) (BETA Google Service)
```

Comment:
it's written this way to specify that the Google service is in Beta, not the SB support for it.